play/src/front/Chat/Connection/Matrix/MatrixChatConnection.ts

```diff
@@ -686,6 +690,18 @@ export class MatrixChatConnection implements ChatConnectionInterface {
             this.deleteRoom(roomId).catch((e) => {
                 console.error("Failed to delete room : ", e);
             });
+            //get room parent folder and refresh his child rooms
+            const parentsFolderIds = this.getParentRoomID(room);
+            void (async () => {
+                try {
+                    const parentFolder = await this.findParentFolder(parentsFolderIds[0]);
+                    if (parentFolder) {
+                        await parentFolder.refreshAllChildRooms();
+                    }
+                } catch (e) {
+                    console.error("Failed to refresh child rooms:", e);
```

Comment:
Could we imagine strong code to try it again or show icon / information to the user that could say "Error refresh, click here or wait 30 sec..."?